src/components/ElectionResultsDiscreteTableSection/ElectionResultsDiscreteTableSection.tsx

```diff
@@ -25,14 +32,22 @@ const CandidateTable: React.FC<{
         </thead>
         <tbody>
           {candidates &&
-            candidates.map((candidate, index) => (
-              <tr key={index}>
-                <td className={classes.nameCell}>{candidate.name}</td>
-                <td>{formatGroupedNumber(candidate.votes)}</td>
-              </tr>
-            ))}
+            candidates.map(
+              (candidate, index) =>
+                (!(canCollapse && collapsed) || index < 5) && (
+                  <tr key={index}>
+                    <td className={classes.nameCell}>{candidate.name}</td>
+                    <td>{formatGroupedNumber(candidate.votes)}</td>
+                  </tr>
+                ),
+            )}
         </tbody>
       </ResultsTable>
+      {canCollapse && (
+        <Button className={classes.collapseButton} onClick={onToggleCollapsed}>
+          {collapsed ? "Afișează toate rezultatele" : "Ascunde toate rezultatele"}
```

Comment:
Are we sure "Ascunde toate rezultatele" is the appropriate message here? It's only hiding the 6th+ results, not everything.

Reply:
Good catch, will change the message